authnz/src/main/java/io/stargate/auth/entity/AuthEntityWrapper.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.auth.entity;
+
+import io.stargate.db.Persistence;
+import org.immutables.value.Value;
+
+/**
+ * This is a base class of authentication and authorization data exposed by a particular {@link
+ * Persistence} implementation.
+ *
+ * <p>The wrapped object is intended to be recoverable by code that is aware of the the persistence
+ * implementation details. Sub-interfaces also define some generic property accessors.
+ *
+ * @param <Wrapped> The persistence-specific entity type.
```

Comment:
I don't see a `Wrapped` class in this PR. What am I missing?

Reply:
It's a type parameter... I'm thinking to remove the wrappers, actually.